### DISCUSSION TITLE
a new frame without extending the previous one

### DISCUSSION THREAD

**Discussion by Scvosh** [keiwando/toonsquid#107] [Ideas]
1 - a new frame without extending the previous one

up to 12 frames is one drawing, then if you start drawing from 30 frames, then the last drawing is automatically extended to 30 frames (you constantly have to create empty keyframes, this greatly slows down the process.)
2 - QuickShape as in (Procreate) + brushes from the same place.

**Comment by keiwando** [comment]
The fact that the previous frame is getting extended even if you have the "magnetic mode" button disabled is a bug, which I will fix for the next update. Thanks for pointing it out!
